{"status":"ERROR","subtests":[{"name":"scale() with a scale factor of zero works","status":"TIMEOUT"}]}